### PR TITLE
installer: use MSVC2015 runtime.

### PR DESCRIPTION
**Pull request by mkrautz**

### CODE REVIEW

**Comment by mkrautz** [comment]
@hacst ptal

File: installer/Files.wxs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+a<?xml version="1.0" encoding="utf-8"?>
```
Comment (mkrautz) [inline installer/Files.wxs]: bad

File: installer/Product.wxs [hunk 2]
```diff
@@ -151,8 +151,8 @@
 
       <?ifdef VersionSubDir ?>
         <?ifdef RedistDirVC12 ?>
```
Comment (hacst) [inline installer/Product.wxs]: s/VC12/VC14

File: installer/Files.wxs [hunk 3]
```diff
@@ -135,12 +135,12 @@
       </Component>
 
       <?ifdef VersionSubDir ?>
-        <?ifdef RedistDirVC12 ?>
-        <Component Id="Murmur_msvcp120.dll">
-          <File Id="Murmur_msvcp120.dll" Source="$(var.RedistDirVC12)\msvcp120.dll" KeyPath="yes" />
+        <?ifdef RedistDirVC14 ?>
+        <Component Id="Murmur_msvcp140.dll">
+          <File Id="Murmur_msvcp140.dll" Source="$(var.RedistDirVC12)\msvcp140.dll" KeyPath="yes" />
```
Comment (hacst) [inline installer/Files.wxs]: *VC14

File: installer/Files.wxs [hunk 4]
```diff
@@ -135,12 +135,12 @@
...
         </Component>
-        <Component Id="Murmur_msvcr120.dll">
-          <File Id="Murmur_msvcr120.dll" Source="$(var.RedistDirVC12)\msvcr120.dll" KeyPath="yes" />
+        <Component Id="Murmur_vcruntime140.dll">
+          <File Id="Murmur_vcruntime140.dll" Source="$(var.RedistDirVC12)\vcruntime140.dll" KeyPath="yes" />
```
Comment (hacst) [inline installer/Files.wxs]: *VC14

**Comment by hacst** [comment]
See comment. LGTM apart from that.

**Pull request merged**